### DISCUSSION TITLE
crash while loading fonts

### DISCUSSION THREAD

**Discussion by HadenGrenade** [ocornut/imgui#6110] [New Users Build/Link/Run issues ONLY!]
Hey everyone, im making my first IMGUI menu, and im having some trouble with this crash. Thanks in advance! 
(The name is blurred out because its a curse word and im not sure if im allowed to say that here)
![Screenshot 2023-01-22 130641](https://user-images.githubusercontent.com/80079690/213932563-4e90d54a-f441-4df2-8919-676aeeae60c1.png)

**Comment by ocornut** [comment]
Sorry but #1586
